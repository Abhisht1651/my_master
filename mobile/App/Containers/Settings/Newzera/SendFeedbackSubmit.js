// @flow

import React from 'react'
import { Text, View, Image, Dimensions, TouchableOpacity } from 'react-native'
import { NavigationContainerProps } from '@react-navigation/native'

import { Images } from '../../../Themes'

import styles from './Styles/SendFeedbackStyle'
// import common from './Styles/CommonStyle'

const Width = Dimensions.get('window').width
const Height = Dimensions.get('window').height

type Props = NavigationContainerProps & {}
export default function SendFeedbackSubmit(props: Props) {
  const { navigation } = props
  return (
    <View>
      <View style={{ paddingTop: (56 / 812) * Height }}>
        <View style={styles.flexRow}>
          <View style={{ paddingTop: (6 / 812) * Height }}>
            <TouchableOpacity onPress={() => navigation.navigate('NewzeraSettings')}>
              <Image source={Images.leftArrowback} style={styles.leftArrow} />
            </TouchableOpacity>
          </View>
          <View style={{ paddingLeft: (73.5 / 375) * Width }}>
            <Text style={styles.heading}>Send Feedback</Text>
          </View>
        </View>
        <View style={{ paddingTop: (31 / 812) * Height }}>
          <View>
            <View style={styles.line} />
          </View>
        </View>
        <View style={{ paddingTop: (246 / 812) * Height }} />
        <View style={{ paddingLeft: (38 / 375) * Width, paddingRight: (38 / 375) * Width }}>
          <Text style={styles.feedback}>Thank you for your feedback!</Text>
        </View>
        <View style={{ paddingTop: (31 / 812) * Height }} />
        <View style={{ paddingLeft: (75 / 375) * Width, paddingRight: (75 / 375) * Width }}>
          <View style={styles.submitAgain}>
            <TouchableOpacity onPress={() => navigation.navigate('SendFeedback')}>
              <View style={{ paddingTop: (16 / 812) * Height }}>
                <Text style={styles.submitAgainText}>Submit Again</Text>
              </View>
            </TouchableOpacity>
          </View>
        </View>
      </View>
    </View>
  )
}
