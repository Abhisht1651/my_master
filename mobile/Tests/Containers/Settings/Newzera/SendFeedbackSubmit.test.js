// @flow
import 'react-native'
import React from 'react'
import { shallow } from 'enzyme'
import SendFeedbackSubmit from '../../../../App/Containers/Settings/Newzera/SendFeedbackSubmit'

describe('SendFeedbackSubmit', () => {
  it('SendFeedbackSubmit component renders correctly', () => {
    const tree = shallow(<SendFeedbackSubmit />)
    expect(tree).toMatchSnapshot()
  })

  it('should render the Text Component', () => {
    const wrapper = shallow(<SendFeedbackSubmit />)
    expect(wrapper.find('Text')).toHaveLength(3)
  })

  it('should render the Image Component', () => {
    const wrapper = shallow(<SendFeedbackSubmit />)
    expect(wrapper.find('Image')).toHaveLength(1)
  })
})
