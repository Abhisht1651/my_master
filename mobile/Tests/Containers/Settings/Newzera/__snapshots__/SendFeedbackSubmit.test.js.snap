// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SendFeedbackSubmit SendFeedbackSubmit component renders correctly 1`] = `
<View>
  <View
    style={
      Object {
        "paddingTop": 92,
      }
    }
  >
    <View
      style={
        Object {
          "flexDirection": "row",
          "paddingLeft": 53.99999999999999,
        }
      }
    >
      <View
        style={
          Object {
            "paddingTop": 9.857142857142856,
          }
        }
      >
        <ForwardRef
          onPress={[Function]}
        >
          <Image
            source={
              Object {
                "default": "",
              }
            }
            style={
              Object {
                "height": 22.17857142857143,
                "width": 36,
              }
            }
          />
        </ForwardRef>
      </View>
      <View
        style={
          Object {
            "paddingLeft": 147,
          }
        }
      >
        <Text
          style={
            Object {
              "color": "#1F1F60",
              "fontFamily": "BananaGrotesk-Bold",
              "fontSize": 32.85714285714286,
              "lineHeight": 44.35714285714286,
            }
          }
        >
          Send Feedback
        </Text>
      </View>
    </View>
    <View
      style={
        Object {
          "paddingTop": 50.92857142857143,
        }
      }
    >
      <View>
        <View
          style={
            Object {
              "borderBottomColor": "#E6ECED",
              "borderBottomWidth": 1,
            }
          }
        />
      </View>
    </View>
    <View
      style={
        Object {
          "paddingTop": 404.14285714285717,
        }
      }
    />
    <View
      style={
        Object {
          "paddingLeft": 76,
          "paddingRight": 76,
        }
      }
    >
      <Text
        style={
          Object {
            "color": "#1F1F60",
            "fontFamily": "BananaGrotesk-Medium",
            "fontSize": 29.571428571428573,
            "lineHeight": 36.142857142857146,
            "textAlign": "center",
          }
        }
      >
        Thank you for your feedback!
      </Text>
    </View>
    <View
      style={
        Object {
          "paddingTop": 50.92857142857143,
        }
      }
    />
    <View
      style={
        Object {
          "paddingLeft": 150,
          "paddingRight": 150,
        }
      }
    >
      <View
        style={
          Object {
            "backgroundColor": "#7B46E4",
            "borderRadius": 63.25,
            "height": 82.14285714285714,
            "width": 450,
          }
        }
      >
        <ForwardRef
          onPress={[Function]}
        >
          <View
            style={
              Object {
                "paddingTop": 26.285714285714285,
              }
            }
          >
            <Text
              style={
                Object {
                  "color": "white",
                  "fontFamily": "BananaGrotesk-Bold",
                  "fontSize": 26.285714285714285,
                  "lineHeight": 29.571428571428573,
                  "textAlign": "center",
                }
              }
            >
              Submit Again
            </Text>
          </View>
        </ForwardRef>
      </View>
    </View>
  </View>
</View>
`;
